apps/meteor/tests/end-to-end/api/27-moderation.ts

```diff
@@ -73,6 +74,66 @@ describe('[Moderation]', function () {
 		});
 	});
 
+	describe('[/moderation.userReports]', () => {
```

Comment:
this endpoint is missing some tests for fails, like missing permission or auth, invalid params, etc.

Reply:
Thank you. I will improve the tests in the next PR.